x-pack/plugins/transform/public/app/sections/create_transform/components/step_details/step_details_form.tsx

```diff
@@ -257,15 +256,9 @@ export const StepDetailsForm: FC<Props> = React.memo(
             label={i18n.translate('xpack.transform.stepDetailsForm.transformDescriptionLabel', {
               defaultMessage: 'Transform description',
             })}
-            helpText={i18n.translate(
-              'xpack.transform.stepDetailsForm.transformDescriptionHelpText',
-              {
-                defaultMessage: 'Optional descriptive text.',
-              }
-            )}
           >
             <EuiFieldText
-              placeholder="transform description"
+              placeholder="Description (optional)"
```

Comment:
```suggestion
              {i18n.translate(
              'xpack.transform.stepDetailsForm.transformDescriptionPlaceholderText',
                { defaultMessage: 'Description (optional)' }
              )}
```

We need to translate the placeholder text, see the suggestion.